<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Preferences</class>
 <widget class="QDialog" name="Preferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>356</width>
    <height>241</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tabGeometry">
      <attribute name="title">
       <string>&amp;Geometry</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="lblX1">
           <property name="text">
            <string>x1: </string>
           </property>
           <property name="buddy">
            <cstring>editX1</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="QLineEdit" name="editX1"/>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="lblY1">
           <property name="text">
            <string>y1: </string>
           </property>
           <property name="buddy">
            <cstring>editY1</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QLineEdit" name="editY1"/>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QLabel" name="lblX2">
           <property name="text">
            <string>x2:</string>
           </property>
           <property name="buddy">
            <cstring>editX2</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLineEdit" name="editX2"/>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="lblY2">
           <property name="text">
            <string>y2: </string>
           </property>
           <property name="buddy">
            <cstring>editY2</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QLineEdit" name="editY2"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="lblIterations">
           <property name="text">
            <string>&amp;Iterations: </string>
           </property>
           <property name="buddy">
            <cstring>editIterations</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="editIterations"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>41</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabFractal">
      <attribute name="title">
       <string>&amp;Fractal</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="rbMandelbrot">
         <property name="text">
          <string>&amp;Mandelbrot set</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="lblExponent">
           <property name="text">
            <string>Exponent</string>
           </property>
           <property name="buddy">
            <cstring>sbExponent</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sbExponent">
           <property name="minimum">
            <number>2</number>
           </property>
           <property name="maximum">
            <number>10</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QRadioButton" name="rbJulia">
         <property name="text">
          <string>&amp;Quadratic Julia set</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="lblCRe">
           <property name="text">
            <string>&amp;Real: </string>
           </property>
           <property name="buddy">
            <cstring>editCRe</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="editCRe"/>
         </item>
         <item>
          <widget class="QLabel" name="lblCIm">
           <property name="text">
            <string>I&amp;mag: </string>
           </property>
           <property name="buddy">
            <cstring>editCIm</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="editCIm"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="ChbColors">
         <property name="text">
          <string>Use colors</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>45</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Preferences</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>231</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Preferences</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>231</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
